Library/Homebrew/cleanup.rb

```diff
@@ -46,22 +111,17 @@ def cleanup_keg(keg)
     def cleanup_logs
       return unless HOMEBREW_LOGS.directory?
       HOMEBREW_LOGS.subdirs.each do |dir|
-        cleanup_path(dir) { dir.rmtree } if prune?(dir, days_default: 14)
+        cleanup_path(dir) { dir.rmtree } if dir.prune?(ARGV.value("prune")&.to_i || 14)
```

Comment:
I think the `14` should become a constant as it's no longer documented by the `days_default` now